2023-06-30 00:57:30,014 - logger_ser - ERROR - app.ocr_type_document():76 - Id not found!
2023-06-30 00:57:30,016 - logger_ser - ERROR - app.ocr_type_document():86 - 
Traceback (most recent call last):
  File "/media/HDD1/hai/api-server-template/server/app.py", line 77, in ocr_type_document
    raise Exception
Exception
2023-06-30 00:57:36,756 - logger_ser - ERROR - app.ocr_type_document():76 - Id not found!
2023-06-30 00:57:36,756 - logger_ser - ERROR - app.ocr_type_document():86 - 
Traceback (most recent call last):
  File "/media/HDD1/hai/api-server-template/server/app.py", line 77, in ocr_type_document
    raise Exception
Exception
2023-06-30 00:57:46,873 - logger_ser -  INFO - app.__handle_request():49 - Success!
2023-06-30 10:43:32,966 - logger_ser -  INFO - app.__handle_request():49 - Success!
